tools/speech_data_explorer/README.md

```diff
@@ -8,19 +8,37 @@ Features:
 - navigation across dataset (sorting, filtering)
 - inspection of individual utterances (waveform, spectrogram, audio player)
 - errors' analysis (Word Error Rate, Character Error Rate, Word Match Rate, Mean Word Accuracy, diff)
+- visually compare ASR models 
 
 Please make sure that requirements are installed. Then run:
 ```
 python data_explorer.py path_to_manifest.json
 ```
 
+To use visual comparison:
```

Comment:
To compare word-level accuracy of two ASR models:

Reply:
+